client/landing/stepper/hooks/use-is-playground-eligible.ts

```diff
@@ -1,5 +1,9 @@
 import config from '@automattic/calypso-config';
 
 export function useIsPlaygroundEligible() {
+	return isPlaygroundEligible();
+}
+
+export function isPlaygroundEligible() {
 	return config.isEnabled( 'onboarding/playground' );
```

Comment:
The `onboarding/playground` feature flag is now set to `true` in all environments. Removing this additional check and the flag from all JSON files is perfectly fine. Checking for the `playground` GET parameter is enough. Despite the Playground onboarding being an experiment, it is not publicized to the end user but is only usable if you manually write the URL.

I think this will clean up the code.

Reply:
Removing in a follow up PR https://github.com/Automattic/wp-calypso/pull/102620